e2e/interchaintestv8/e2esuite/suite.go

```diff
@@ -126,3 +126,11 @@ func (s *TestSuite) SetupSuite(ctx context.Context) {
 		s.proposalIDs[chain.Config().ChainID] = 1
 	}
 }
+
+// TODO: Remove when we have removed manual relaying and moved to
+// common relaying methods that can choose the correct methods
+func (s *TestSuite) SkipIfEthTestnetType(testnetType string) {
```

Comment:
Not all the e2e relaying tests work with both pow and pos, so added this for now.
I think it might make sense to get rid of the manual relaying in go and switch to the relayer now, but we can still have the relayer calls behind some functions that can handle the different testnet types - and then we can remove this method.

If we agree on this, I'll create an issue to remove the manual relaying and consolidate relaying in the e2e tests.

Reply:
I still think consolidation the tests would be useful, because I don't want to maintain the manual relayer and a bunch of extra tests if we don't have to. If we can just test everything together it would mean quite a bit less code.

Reply:
But I do like the mock option a lot! :)

Reply:
https://github.com/cosmos/solidity-ibc-eureka/issues/192